### ISSUE TITLE
Getting "TemplateKitError.tag:set: Unsupported key type" after upgrading to Xcode 10.2

### ISSUE THREAD

**Issue by elveatles** [vapor/vapor#1938]
I am still new to Vapor, so I'm sorry if the answer is obvious.

My project was working with Xcode 10.1. After upgrading to 10.2, I can no longer get my web pages to render without errors.

I had a simple website working using Vapor, Fluent with PostgreSQL, and Leaf. After upgrading from Xcode 10.1 to 10.2 then visiting http://localhost:8080/ in Chrome, I get the error: 

> [ ERROR ] TemplateKitError.tag:set: Unsupported key type
> (<Project Path>/Resources/Views/testIndex.leaf line: 0 column: 1 range: 1..<37)
> (TagContext.swift:54)

I can't figure out what's wrong.

### Steps to reproduce

Sorry if this is a lot of code to look at. I'm including everything just in case.

Package.swift:

```swift
// swift-tools-version:4.0
import PackageDescription

let package = Package(
    name: "TILApp",
    dependencies: [
        // 💧 A server-side Swift web framework.
        .package(url: "https://github.com/vapor/vapor.git", from: "3.0.0"),
        // 🔵 Swift ORM (queries, models, relations, etc) built on PostgreSQL.
        .package(url: "https://github.com/vapor/fluent-postgresql.git", from: "1.0.0"),
        .package(url: "https://github.com/vapor/leaf.git", from: "3.0.0")
    ],
    targets: [
        .target(name: "App", dependencies: ["FluentPostgreSQL", "Vapor", "Leaf"]),
        .target(name: "Run", dependencies: ["App"]),
        .testTarget(name: "AppTests", dependencies: ["App"])
    ]
)
```

configure.swift

```swift
import FluentPostgreSQL
import Vapor
import Leaf

/// Called before your application initializes.
public func configure(_ config: inout Config, _ env: inout Environment, _ services: inout Services) throws {
    // Register providers first
    try services.register(FluentPostgreSQLProvider())
    try services.register(LeafProvider())

    // Register routes to the router
    let router = EngineRouter.default()
    try routes(router)
    services.register(router, as: Router.self)

    // Register middleware
    var middlewares = MiddlewareConfig() // Create _empty_ middleware config
    middlewares.use(FileMiddleware.self) // Serves files from `Public/` directory
    middlewares.use(ErrorMiddleware.self) // Catches errors and converts to HTTP response
    services.register(middlewares)

    var databases = DatabasesConfig()
    let databaseConfig = PostgreSQLDatabaseConfig(
        hostname: "localhost",
        port: 5432,
        username: "vapor",
        database: "vapor",
        password: "password")
    let database = PostgreSQLDatabase(config: databaseConfig)
    databases.add(database: database, as: .psql)
    services.register(databases)

    // Configure migrations
    var migrations = MigrationConfig()
    migrations.add(model: TestUser.self, database: .psql)
    services.register(migrations)
    
    var commandConfig = CommandConfig.default()
    commandConfig.useFluentCommands()
    services.register(commandConfig)
    
    config.prefer(LeafRenderer.self, for: ViewRenderer.self)
}
```

routes.swift

```swift
/// Register your application's routes here.
public func routes(_ router: Router) throws {
    let websiteController = WebsiteController()
    try router.register(collection: websiteController)
}
```

WebsiteController.swift

```swift
import Vapor
import Leaf

struct WebsiteController: RouteCollection {
    func boot(router: Router) throws {
        router.get(use: testIndexHandler)
    }

    func testIndexHandler(_ req: Request) throws -> Future<View> {
        let testContext = TestContext()
        return try req.view().render("testIndex", testContext)
    }
}

struct TestContext: Encodable {
    let title = "Test"
}
```

testBase.leaf

```html
<!DOCTYPE html>
<html lang="en">
    <head>
        <!-- Required meta tags -->
        <meta charset="utf-8">
        <title>#(title)</title>
    </head>
    <body>
        #get(content)
    </body>
</html>
```

testIndex.leaf

```html
#set("content") {
    <h1>Test</h1>
}

#embed("testBase")
```

I run the postgres image with docker.

`docker run --name postgres -e POSTGRES_DB=vapor -e POSTGRES_USER=vapor -e POSTGRES_PASSWORD=password -p 5432:5432 -d postgres`

I run my website in Xcode by choosing  Run > My Mac  then clicking the play button.

I see the usual migration messages, then

> Server starting on http://localhost:8080

I go to the URL, then I get the TemplateKitError I pasted above.

### Expected behavior

The webpage should show a simple webpage that just says "Test" in an h1 tag.

### Actual behavior

I see a TemplateKitError described above. No web page rendered. Just json with an error.

### Environment

* Vapor Framework version: 3.2.2
* Vapor Toolbox version: 3.1.10
* OS version: macOS Mojave 10.14.3

**Comment by 0xTim** [comment]
@elveatles This has been fixed in [TemplateKit 1.1.2](https://github.com/vapor/template-kit/releases/tag/1.1.2) - if you run `swift package update` in your project directory you'll update all your dependencies and pull in the fix. (It should also remove all the warnings as well!)

**Comment by elveatles** [comment]
@0xTim Ah jeez. I knew it would be something obvious. Thank you for the fix.
